src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -64,6 +69,7 @@ const ReimbursementRequestTable = ({
                 <TableCell align="center">{dateUndefinedPipe(row.dateSubmittedToSabo)}</TableCell>
                 <TableCell align="center">{row.vendor.name}</TableCell>
                 <TableCell align="center">{cleanReimbursementRequestStatus(row.status)}</TableCell>
+                <TableCell align="center">{cleanReimbursementRequestRefundSource(row.refundSource)}</TableCell>
```

Comment:
the status column should stay on the far right, this refundSource column can go to the left of it